accepted/0000-expand-list-of-packlist-ignored-files.md

```diff
@@ -0,0 +1,37 @@
+# Expand list of ignored files
+
+## Summary
+
+Let's expand the default list of ignored files in [packlist](https://github.com/npm/npm-packlist).
+
+## Motivation
+
+A large population of npm users are concerned about package sizes and with the advent of the file explorer now available on [npmjs.com](https://www.npmjs.com/) we can now see a number of common files that are very intrinsic to the JS community that we could start ignoring from package bundles without too much friction to the larger ecosystem.
+
+## Detailed Explanation
+
+Expand the current list of ignored files to also ignore by default:
```

Comment:
Do the items listed here mean e.g. `/.editorconfig` or `**/.editorconfig`? The latter could cause breakage in generators / initializers that have template files in subdirectories, for example https://github.com/pkgjs/create/tree/main/templates